repos/re-theme/src/dimensions/sizeMap.js

```diff
@@ -141,13 +141,20 @@ export const getSize = width => {
  * @function
  * @param {string} key - Name of the size
  *
- * @returns {Array} - Array of size key names
+ * @returns {Array} - [
+ *   Array of size key names,
+ *   Array of unused size key names
+ * ]
+ *
+ * @example
+ * const [ keys, unused ] = getMergeSizes('$medium')
+ * console.log(keys) -> [ '$medium', '$small', '$xsmall' ]
+ * console.log(unused) -> [ '$large' ]
  */
 export const getMergeSizes = key => {
-  // Add 1 because slice does not include the last item of the range
-  return sizeMap.entries
-    .slice(0, sizeMap.indexes[key] + 1)
-    .map(([ name, size ]) => name)
+  const keys = sizeMap.entries.map(([name]) => name)
```

Comment:
Couldn't you just do `Object.keys(sizeMap.hash)` ?

Reply:
Ahh I meant to do this, but I forgot. Updated